components/central-logger/org.wso2.carbon.identity.central.log.mgt/src/main/java/org/wso2/carbon/identity/central/log/mgt/hanlder/CentralLogger.java

```diff
@@ -43,9 +42,7 @@ public void handleEvent(Event event) {
         switch (eventName) {
             case PUBLISH_AUDIT_LOG:
                 // Publish new audit logs only if the old audit log publishing is disabled.
-                if (isLegacyAuditLogsDisabled()) {
-                    CarbonUtils.publishAuditLogs(eventProperties);
-                }
+                CarbonUtils.publishAuditLogs(eventProperties);
```

Comment:
Is that comment in line 44 required? Since the old audit log disabled check is now done at the event triggering point, IMO this comment is not valid in the event handler

Reply:
fixed